src/library_glfw.js

```diff
@@ -1160,9 +1169,14 @@ var LibraryGLFW = {
     Module["canvas"].addEventListener('drop', GLFW.onDrop, true);
     Module["canvas"].addEventListener('dragover', GLFW.onDragover, true);
 
-    Browser.resizeListeners.push((width, height) => {
-       GLFW.onCanvasResize(width, height);
+#if USE_GLFW == 3
+    Browser.isHiDPIAware = !!Module['isHiDPIAware'] || false;
```

Comment:
If you indent this to be part of the incoming module API then can you add it to `EXTRA_INCOMING_JS_API` in emcc.py and use `{{{ makeModuleReceiveExpr(...) }}}` here?

Reply:
I am not sure if I am doing something wrong, but adding `isHiDPIAware` to `EXTRA_INCOMING_JS_API` then compiling the code generates this:

```javascript
function checkIncomingModuleAPI() {
  ignoredModuleProp('fetchSettings');
  ignoredModuleProp('isHiDPIAware');
}
```

which fails with this message:

```text
Aborted(`Module.isHiDPIAware` was supplied but `isHiDPIAware` not included in INCOMING_MODULE_JS_API)
```

Here is the generated code for `ignoredModuleProp`

```javascript
function ignoredModuleProp(prop) {
  if (Object.getOwnPropertyDescriptor(Module, prop)) {
    abort(`\`Module.${prop}\` was supplied but \`${prop}\` not included in INCOMING_MODULE_JS_API`);
  }
}
```
Any idea?

Reply:
You would need to this specify `-sINCOMING_MODULE_JS_API=isHiDPIAware` on the command line.

Reply:
Thank you. I am just wondering if it is necessary anymore as it seems a bit complicated for the end user. 

When I started working on the changes that is the idea I came up with because I needed a way to change this flag. Since I added a `setHiDPIAware` on `Browser` which gets injected into `Module` (similarly to `requestFullscreen`, `createContext`, etc...), the calling code can easily make that determination. For example, in the resizable demo I linked to, I have a ImGui checkbox which changes this flag using the API...

So I think I am simply going to remove this line entirely unless you have a different opinion.

Reply:
sgtm